browser/brave_news/brave_news_tab_helper.cc

```diff
@@ -60,14 +66,9 @@ const std::vector<GURL> BraveNewsTabHelper::GetAvailableFeedUrls() {
   base::flat_set<GURL> seen_feeds;
 
   auto current_url = GetWebContents().GetLastCommittedURL();
-  if (!current_url.is_empty() && !current_url.host().empty()) {
-    auto* default_publisher =
-        controller_->publisher_controller()->GetPublisherForSite(current_url);
```

Comment:
Q. I'm curious why we don't get default publishers anymore here.

Reply:
Replaced  `page_feed_` - we have to request it asynchronously now